src/_includes/components/projects-grid.html

```diff
@@ -0,0 +1,16 @@
+<section class="projects-grid">
+  <div class="projects-grid__tag-nav">
+    {% for project in site.projects %}
+      <a href="#">
+        {{ project.title }}
+      </a>
+    {% endfor %}
+    </span>
+  </div>
+
+  <div class="projects-grid__items">
+    {% for project in site.projects %}
+      {% include components/projects-grid-item.html project=project %}
```

Comment:
I want to filter by tag here. There's probably a ruby way to write that for loop, that would pre-filter it of projects not including the selected tag.

It would be nice to store the selected tag as a query parameter. If you know an easy way to do that please let me know.

Reply:
You could approach it that way, but it will be painful to manage it. It would require us to generate a set of pages based on the query params. The way we're going to proceed with this is like the Canswe retailers page. Use js to filter and sort the list of projects.